# 19 Refinery utilization

On weeks when refinery utilization is further from its pre-2020 median than the train 90th percentile of that distance, is the holdout mean absolute Wednesday RWTC return larger?

Holdout starts 2020-01-01. Verdict: held.

WPULEUS3 minus its train median, in percentage points of utilization. Train median 90.40 percent. An anomaly is an absolute deviation at or above the train 90th percentile of absolute deviations, frozen at 7.10 points.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'util-anomaly')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


held
Train gap in mean absolute Wednesday return, anomaly minus the rest, is 0.264%. Holdout gap is 0.787% (62 anomalies, 286 other weeks). The holdout gap is positive, which was the bar.
Mean absolute Wednesday RWTC return on utilization anomalies
Train | 137 | 1193 | 0.264%
Holdout | 62 | 286 | 0.787%


## Method

The bar is a positive holdout gap in absolute return. The cutoff stays at the train 90th percentile.

## Result

Train gap in mean absolute Wednesday return, anomaly minus the rest, is 0.264%. Holdout gap is 0.787% (62 anomalies, 286 other weeks). The holdout gap is positive, which was the bar.